tests/pipeline/test_clean.py

```diff
@@ -0,0 +1,92 @@
+from pathlib import Path
+from unittest.mock import AsyncMock, MagicMock
+
+import pytest
+from pytest_mock import MockerFixture
+from typer.testing import CliRunner
+
+from kpops.cli.main import app
+
+runner = CliRunner()
+
+RESOURCE_PATH = Path(__file__).parent / "resources"
+
+
+@pytest.mark.usefixtures("mock_env", "load_yaml_file_clear_cache")
+class TestClean:
+    @pytest.fixture(autouse=True)
+    def helm_mock(self, mocker: MockerFixture) -> MagicMock:
+        async_mock = AsyncMock()
+        return mocker.patch(
+            "kpops.components.base_components.helm_app.Helm",
+            return_value=async_mock,
+        ).return_value
+
+    # TODO: test using public Pipeline API
+    # @pytest.fixture()
+    # def config(self) -> KpopsConfig:
+    #     return KpopsConfig()
+
+    # @pytest.fixture()
+    # def handlers(self) -> ComponentHandlers:
+    #     return ComponentHandlers(
+    #         schema_handler=AsyncMock(),
+    #         connector_handler=AsyncMock(),
+    #         topic_handler=AsyncMock(),
+    #     )
+
+    # @pytest.fixture()
+    # def pipeline(self, config: KpopsConfig, handlers: ComponentHandlers) -> Pipeline:
+    #     kwargs = {"config": config, "handlers": handlers}
+    #     pipeline = Pipeline()
+    #     pipeline.add(
+    #         ProducerApp(
+    #             name="producer",
+    #             namespace="test-namespace",
+    #             app=ProducerAppValues(
+    #                 streams=ProducerStreamsConfig(brokers=config.kafka_brokers)
+    #             ),
+    #             **kwargs,
+    #         )
+    #     )
+    #     return pipeline
```

Comment:
Is this intentionally left like this?

Reply:
yep, we don't have this part of the Pipeline API (deploy, destroy, clean, ...) yet but I am planning to add it soon